packages/app/components/trending/tabs/nfts-list.tsx

```diff
@@ -1,33 +1,19 @@
-import {
-  useCallback,
-  forwardRef,
-  useImperativeHandle,
-  useRef,
-  useMemo,
-} from "react";
-import { StyleSheet } from "react-native";
+import { useCallback, forwardRef, useImperativeHandle, useRef } from "react";
 
 import type { ListRenderItemInfo } from "@shopify/flash-list";
-import chunk from "lodash/chunk";
 
 import { useRouter } from "@showtime-xyz/universal.router";
 import { TabInfiniteScrollList } from "@showtime-xyz/universal.tab-view";
-import { View } from "@showtime-xyz/universal.view";
 
-import { Card } from "app/components/card";
+import { Card, GAP } from "app/components/card";
 import { ListFooter } from "app/components/footer/list-footer";
-import { withViewabilityInfiniteScrollList } from "app/hocs/with-viewability-infinite-scroll-list";
 import { useTrendingNFTS } from "app/hooks/api-hooks";
 import { useContentWidth } from "app/hooks/use-content-width";
 import { useScrollToTop } from "app/lib/react-navigation/native";
 import { NFT } from "app/types";
 
 import { TrendingTabListProps, TrendingTabListRef } from "./";
 
-const ViewabilityInfiniteScrollList = withViewabilityInfiniteScrollList(
```

Comment:
remove this, I think we shouldn‘t need this in the Trending list.

Reply:
I think that's wrong. I have seen that we're playing videos in web, even in the trending list. So it might have its purpose.

Reply:
yeah, I noticed that, but looks like we didn't using this on profile list. 😃

Reply:
anyways, adding it at least should be better I think.

Reply:
haha, just I found this list didn't use on the web, accurately now using [this file](https://github.com/showtime-xyz/showtime-frontend/blob/staging/packages/app/components/trending/trending.web.tsx) on the web.

Reply:
Oh ok :)